vue-components/tests/unit/components/Lookup.spec.ts

```diff
@@ -135,15 +148,101 @@ describe( 'Lookup', () => {
 			{ label: 'potato', description: 'root vegetable' },
 			{ label: 'duck', description: 'aquatic bird' },
 		];
-		const wrapper = mount( Lookup, { propsData: { menuItems } } );
-		const inputField = wrapper.findComponent( Input );
-		inputField.setValue( 'something something' );
-		inputField.trigger( 'focus' );
+		const wrapper = createLookupWrapperWithExpandedMenu( menuItems );
 
 		return Vue.nextTick().then( () => {
 			expect( wrapper.findComponent( LookupMenu ).props( 'menuItems' ) ).toBe( menuItems );
-
 		} );
 	} );
 
+	it( 'emits an `input` event containing the selected menu item upon selection', async () => {
+		const menuItems = [
+			{ label: 'potato', description: 'root vegetable' },
+			{ label: 'duck', description: 'aquatic bird' },
+		];
+		const wrapper = createLookupWrapperWithExpandedMenu( menuItems );
+
+		await Vue.nextTick();
+
+		const selectedItem = 1;
+		wrapper.findAll( '.wikit-LookupMenu__item' ).at( selectedItem ).element.click();
+
+		expect( wrapper.emitted( 'input' )![ 0 ] ).toEqual( [ menuItems[ selectedItem ] ] );
+	} );
+
+	it( "emits `update:searchInput` containing the selected item's label upon selection", async () => {
+		const menuItems = [
+			{ label: 'potato', description: 'root vegetable' },
+			{ label: 'duck', description: 'aquatic bird' },
+		];
+		const wrapper = createLookupWrapperWithExpandedMenu( menuItems );
+
+		await Vue.nextTick();
+
+		const selectedItem = 1;
+		wrapper.findAll( '.wikit-LookupMenu__item' ).at( selectedItem ).element.click();
+
+		expect( ( wrapper.emitted( 'update:searchInput' ) )![ 0 ] )
+			.toEqual( [ menuItems[ selectedItem ].label ] );
+	} );
+
+	it( 'emits `update:searchInput` when the internal input element receives input', () => {
+		const wrapper = mount( Lookup );
+		const userInput = 'potato';
+		wrapper.find( 'input' ).setValue( userInput );
+
+		expect( ( wrapper.emitted( 'update:searchInput' ) )![ 0 ] ).toEqual( [ userInput ] );
+	} );
+
+	it( 'checks if the menu closes on Esc key', () => {
```

Comment:
A note about the naming of tests here. It should describe what the system under test does, not the test itself. E.g.:

```suggestion
	it( 'closes the menu on Esc key', () => {
```